components/renku_data_services/app_config/config.py

```diff
@@ -9,120 +9,43 @@
 instantiated multiple times without creating multiple database connections.
```

Comment:
issue: This whole docstring is not really applicable now. As far as I can see we dont do any of the `@property` stuff now in this specific file.